projects/wavelet_dataAggregation/checkDataType.py

```diff
@@ -0,0 +1,188 @@
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2016, Numenta, Inc.  Unless you have an agreement
+# with Numenta, Inc., for a separate license for this software code, the
+# following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero Public License for more details.
+#
+# You should have received a copy of the GNU Affero Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+
+from os import listdir
```

Comment:
A few minor things, style-wise in this file:
* re-order imports. See https://github.com/numenta/nupic/wiki/Python-Style-Guide#import-grouping
* Limit line length to 80 characters
* Blank lines: https://github.com/numenta/nupic/wiki/Python-Style-Guide#blank-lines (between top level functions = 3 blank lines. Between methods inside classes = 2 blank lines)